crates/kitsune_p2p/kitsune_p2p/src/spawn/actor/space.rs

```diff
@@ -372,9 +367,12 @@ async fn update_single_agent_info(
         network_type,
         mdns_handles,
         bootstrap_service,
+        resize_arcs,
     } = input;
 
-    update_arc_length(evt_sender, space.clone(), &mut arc).await?;
+    if resize_arcs {
```

Comment:
I guess this was already done?

Reply:
Ah sorry @freesig didn't realize this was exactly what you meant by plumbing this config through